frontend/src/components/SamplesList.tsx

```diff
@@ -450,14 +450,6 @@ export default function SamplesList({
                 locked: function (params) {
                   return params.data?.revisable === false;
                 },
-                "validation-error": function (params) {
-                  return (
-                    params.data?.revisable === true &&
-                    (params.data?.validationStatus === false ||
-                      (params.data?.validationStatus === null &&
-                        params.data?.sampleCategory !== "clinical"))
-                  );
-                },
```

Comment:
Remove the red background when a sample has a validation warning to be consistent with the Requests page. (The UX team also recommended removing this so might as well.)